src/main/java/pl/stormit/eduquiz/quizcreator/domain/quiz/Quiz.java

```diff
@@ -29,6 +32,8 @@ public class Quiz {
     @GeneratedValue
     private UUID id;
 
+    @NotBlank(message = "Quiz Name must not be blank")
+    @Size(min = 2, max = 20)
```

Comment:
I think, maximum value should be greater - e.g. we have quiz name "General knowledge of world history" in database.
Maybe we should use 255 characters?